measure.sh

```diff
@@ -28,8 +28,14 @@ wait_for_query_value() {
 
     until [ "$result" -eq 1 ]
     do
-        sleep 30
-        value=$(curl -s $url -d "query=$1" -H "Authorization: Bearer $token" | jq '.data.result[0].value[1] | tonumber')
+        sleep 10
+        value=$(curl -s $url -d "query=$1" -H "Authorization: Bearer $token" | jq ' try .data.result[0].value[1] | tonumber')
+        until [ ! -z "$value" ]
```

Comment:
:question: Can this loop forever? :smile: We have a timeout on the job in general, so it's probably fine I guess :shrug:

Reply:
Humm perhaps having a limit it could be waiting is better, I will add that :)